virt/esx/esx.py

```diff
@@ -310,7 +312,7 @@ def getHostGuestMapping(self):
                     # Stopped machine doesn't have any uuid
                     if vm.uuid is not None:
                         l.append(vm.uuid)
-                mapping[host.uuid] = l
+                mapping[host.name] = l
```

Comment:
is 'escxli system uuid get' the instanceUuid ( https://www.vmware.com/support/developer/converter-sdk/conv43_apireference/vim.vm.ConfigInfo.html ) ?

The uuid virt-who is getting is the VirtualMachineConfig.uuid instead.